awspub/image.py

```diff
@@ -500,7 +508,20 @@ def _register_images(self, snapshot_ids: Dict[str, str], region: str, ec2client_
         if self.conf["billing_products"]:
             register_image_kwargs["BillingProducts"] = self.conf["billing_products"]
 
-        resp = ec2client_region.register_image(**register_image_kwargs)
+        try:
+            resp = ec2client_region.register_image(**register_image_kwargs)
+        except botocore.exceptions.ClientError as e:
+            if e.response.get("Error", {}).get("Code", None) == "OperationNotPermitted":
+                logger.warning(
```

Comment:
shouldn't we use `logger.exception()` here?

Reply:
yep, good catch!